cartridges/plugins/wso2am-1.9.0/wso2am-startup-handler.py

```diff
@@ -0,0 +1,440 @@
+            #                                       self.ENV_CONFIG_PARAM_KEYMANAGER_IP)
+            keymanager_ip = self.get_data_from_meta_data_service(app_id, self.ENV_CONFIG_PARAM_KEYMANAGER_IP)
+            self.export_env_var(self.ENV_CONFIG_PARAM_KEYMANAGER_IP, keymanager_ip)
+            # self.export_env_var(self.ENV_CONFIG_PARAM_THRIFTSERVERHOST, keymanager_ip)
+
+            member_ip = socket.gethostbyname(socket.gethostname())
+            self.update_hosts_file(member_ip, worker_hostname)
+
+            start_command = "exec ${CARBON_HOME}/bin/wso2server.sh -Dprofile=gateway-worker start"
+
+        elif profile == self.CONST_PUBLISHER:
+
+            isKubernetes = self.check_for_kubernetes_cluster(service_name, app_id)
+            service_ip = self.configure_service_ip(load_balancer_ip, isKubernetes, service_name, app_id)
+            store_ip = self.configure_profile(app_id, self.ENV_CONFIG_PARAM_PUBLISHER_IP, service_ip,
+                                              self.ENV_CONFIG_PARAM_STORE_IP)
+            wka_member_value = "[" + store_ip + ":4000]"
+            self.export_env_var(self.ENV_CONFIG_PARAM_STORE_IP, store_ip)
+            self.export_env_var(self.ENV_CONFIG_PARAM_WKA_MEMBERS, wka_member_value)
+
+            keymanager_ip = self.get_data_from_meta_data_service(app_id, self.ENV_CONFIG_PARAM_KEYMANAGER_IP)
+            gateway_ip = self.get_data_from_meta_data_service(app_id, self.ENV_CONFIG_PARAM_GATEWAY_IP)
+            self.export_env_var(self.ENV_CONFIG_PARAM_KEYMANAGER_IP, keymanager_ip)
+            self.export_env_var(self.ENV_CONFIG_PARAM_GATEWAY_IP, gateway_ip)
+            start_command = "exec ${CARBON_HOME}/bin/wso2server.sh -Dprofile=api-publisher start"
+
+        elif profile == self.CONST_STORE:
+
+            isKubernetes = self.check_for_kubernetes_cluster(service_name, app_id)
+            service_ip = self.configure_service_ip(load_balancer_ip, isKubernetes, service_name, app_id)
+            publisher_ip = self.configure_profile(app_id, self.ENV_CONFIG_PARAM_STORE_IP, service_ip,
+                                                  self.ENV_CONFIG_PARAM_PUBLISHER_IP)
+            wka_member_value = "[" + publisher_ip + ":4000]"
+            self.export_env_var(self.ENV_CONFIG_PARAM_PUBLISHER_IP, publisher_ip)
+            self.export_env_var(self.ENV_CONFIG_PARAM_WKA_MEMBERS, wka_member_value)
+
+            keymanager_ip = self.get_data_from_meta_data_service(app_id, self.ENV_CONFIG_PARAM_KEYMANAGER_IP)
+            gateway_ip = self.get_data_from_meta_data_service(app_id, self.ENV_CONFIG_PARAM_GATEWAY_IP)
+            self.export_env_var(self.ENV_CONFIG_PARAM_KEYMANAGER_IP, keymanager_ip)
+            self.export_env_var(self.ENV_CONFIG_PARAM_GATEWAY_IP, gateway_ip)
+            start_command = "exec ${CARBON_HOME}/bin/wso2server.sh -Dprofile=api-store start"
+
+        else:
+            start_command = "exec ${CARBON_HOME}/bin/wso2server.sh start"  # start configurator
```

Comment:
Might be better to describe what this else block would mean. Is it starting the server in default mode?